index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="./style.css" />
+		<title>Awesome books</title>
+	</head>
+	<body>
+		<nav>
+			<div>Awesome Books</div>
+			<ul class="nav-ul">
+				<li class="nav-link" id="all-books"><a href="#all-books">List</a></li>
+				<li class="line"></li>
+				<li class="nav-link" id="add-book"><a href="#add-book">Add new</a></li>
+				<li class="line"></li>
+				<li class="nav-link" id="contact-section"><a href="#contact-section">Contact</a></li>
+			</ul>
+		</nav>
+		<div class="date"></div>
+		<section class="all-books all hidden">
+			<h2>All awesome books</h2>
+			<section class="books"></section>
+		</section>
```

Comment:
- [ OPTIONAL] Be careful with blank lines and indentation. Do not add blank lines, spaces, or indentations without a reason, for readability, add blank lines to separate large or logical code blocks.[ HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)